proto/ssz_proto_library.bzl

```diff
@@ -15,11 +15,14 @@ mainnet = {
     "previous_epoch_attestations.max": "4096",  # MAX_ATTESTATIONS * SLOTS_PER_EPOCH
     "current_epoch_attestations.max": "4096",  # MAX_ATTESTATIONS * SLOTS_PER_EPOCH
     "slashings.size": "8192",  # EPOCHS_PER_SLASHINGS_VECTOR
-    "sync_committee_bits.size": "512", #SYNC_COMMITTEE_SIZE
+    "sync_committee_bits.size": "512",  #SYNC_COMMITTEE_SIZE
```

Comment:
remove space

Reply:
IDE auto did this.   (Let's keep it, otherwise, it will do the same again when modify another time)

Reply:
Can you change the IDE config? I think it's better to minimize changes

Reply:
Yes, for this file (.bzl) I removed the extra white space to match the original file. 

   - Within this file, looks like there are inconsistent comment style (with / without leading space already).  
   
   - For code review, I found an option to "Hide whitespace", in the setting icon near the top left.

Reply:
I would just avoid the change if possible, think also about conflicts etc. Maybe disable the feature in your editor for the time being?